src/NServiceBus.Core/Serializers/XML/XmlDeserialization.cs

```diff
@@ -284,7 +284,7 @@ Type InferNodeType(XmlNode node, object parent)
                 }
             }
 
-            throw new Exception("Could not determine type for node: '" + node.Name + "'.");
+            throw new Exception($"Could not determine type for node: \'{node.Name}\'.");
```

Comment:
Looks these "\" got missed. If I have a second, I'll try an push up a fix for it.